worker.config.json

```diff
@@ -4,6 +4,6 @@
         "extensions": [".jar"],
         "defaultExecutablePath": "%JAVA_HOME%/bin/java",
         "defaultWorkerPath": "azure-functions-java-worker.jar",
-        "arguments": ["-XX:+TieredCompilation -XX:TieredStopAtLevel=1 -noverify -Djava.net.preferIPv4Stack=true -jar", "%JAVA_OPTS%"]
+        "arguments": ["-XX:+TieredCompilation -XX:TieredStopAtLevel=1 -noverify -Djava.net.preferIPv4Stack=true -jar", "%JAVA_OPTS%", "%MESH_JAVA_OPTS%"]
```

Comment:
Thank you sir, can we name I something related to AZURE_FUNCTIONS_MESH_JAVA_OPTS?
Also did we test this change with existing JAVA_OPTS and make sure that both will be picked with the host?

Reply:
I've done that. I also finish testing both on windows and mesh images